README.md

```diff
@@ -24,15 +23,12 @@ Advanced features of Taichi include [spatially sparse computing](https://taichi.
 
 <a href="https://github.com/taichi-dev/taichi/blob/master/examples/mpm_lagrangian_forces.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/lagrangian.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/pbf2d.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/pbf.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/game_of_life.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/game_of_life.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/euler.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/euler.gif" height="192px"></a>
 
-
-## Installation [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
+## Installation [![Downloads](https://img.shields.io/pypi/dw/taichi)](https://pepy.tech/project/taichi)
 
 ```bash
 python3 -m pip install taichi
 ```
 
-**News: (Apr 29, 2020) The OpenGL Compute Shader backend by [彭于斌 (archibate)](https://github.com/archibate)** is officially released in [**v0.6.0**](https://github.com/taichi-dev/taichi/releases/tag/v0.6.0)! Please try it out with `ti.init(arch=ti.opengl)`.
```

Comment:
Removing this because it's not a news now (Apr 29 vs Mar 13).
I guess it's better to replace this with:
```
**News: (Mar 17, 2020) TaichiCon... will be held on... EST... CST... issue page = ...
```
WDYT?

Reply:
Thanks for the proposal. Personally I think README is for everyone around the world, and TaichiCon 0 will be for Chinese developers only. Let's still put OpenGL there (which is a great accomplishment), but update when we host TaichiCon 1 in English.